repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -1484,11 +1486,13 @@ private boolean getAppendOptionForRelationship(AtlasVertex entityVertex, String
         boolean                             ret                       = false;
         String                              entityTypeName            = AtlasGraphUtilsV2.getTypeName(entityVertex);
         AtlasEntityDef                      entityDef                 = typeRegistry.getEntityDefByName(entityTypeName);
-        List<AtlasRelationshipAttributeDef> relationshipAttributeDefs = entityDef.getRelationshipAttributeDefs();
+        if (entityDef != null) {
```

Comment:
This check is not required since we handling in the method above.
Adding this check might result in silent failures.